### PR TITLE
feat: Add control on draft or open status through a param

### PR DESCRIPTION
**Pull request by Kontrolix**
Mergify-cli stack change-Id: I35eff5ee0da1b892deaa3ad5a73e168ccf064032:

### CODE REVIEW

**Comment by Kontrolix** [comment]
This pull request is part of a stack:
1. chore: rename git_push_stack to mergify_cli ([#162](https://github.com/Mergifyio/mergify-cli/pull/162))
1. fix: Make worktree compliant ([#163](https://github.com/Mergifyio/mergify-cli/pull/163))
1. feat: get token through `gh auth token` if not provided by --token or GITHUB_TOKEN env var ([#164](https://github.com/Mergifyio/mergify-cli/pull/164))
1. fix: convert coroutine to task ([#165](https://github.com/Mergifyio/mergify-cli/pull/165))
1. feat: Add control on draft or open status through a param ([#166](https://github.com/Mergifyio/mergify-cli/pull/166))
1. feat: Define trunk through param --trunk/-t or git config mergify-cli.stack-trunk ([#167](https://github.com/Mergifyio/mergify-cli/pull/167))
1. fix: Improve hook ([#168](https://github.com/Mergifyio/mergify-cli/pull/168))
1. chore: remove useless param --stack ([#169](https://github.com/Mergifyio/mergify-cli/pull/169))
1. docs: reademe ([#170](https://github.com/Mergifyio/mergify-cli/pull/170))

File: mergify_cli/__init__.py [hunk 1]
```diff
@@ -560,12 +570,14 @@ def get_default_token() -> str:
 
 def cli() -> None:
     global DEBUG
+    global DRAFT
```
Comment (sileht) [inline mergify_cli/__init__.py:573]: That should not be a global. (yeah the DEBUG way is clearly a bad practice)

Comment (sileht) [reply mergify_cli/__init__.py:573]: why not just pass the args.draft around where this is needed?

Comment (Kontrolix) [reply mergify_cli/__init__.py:573]: I don't find that really handy to carry this param in every function that needed it even more if we refactor everything in the near (or not so far) future. And even if the global style is a bad practice, in this case, it's not really harmful it's not buried in N layers of dependencies.

**Pull request merged**